src/main/resources/static/api.yml

```diff
@@ -997,6 +1032,154 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /assessments:
+    get:
+      tags:
+        - Assessment data
+      summary: Gets assessments the user is authorised to view
+      parameters:
+        - in: query
+          name: filter
+          required: false
+          description: all (default) returns all assessments the user is authorised to view.  Allocated returns only those allocated to the user.
```

Comment:
I wonder whether an alternative could be:

```
GET /assessments - return all assessments the user is allowed to view
GET /assessments?probationOfficerId=some-uuid
```

When a "normal" user is logged in, it's likely that the only assessments they're allowed to see are the ones they're allocated to. Adding the `probationOfficerId` query string is then a way for "admin" users to filter by a particular probation officer.

Reply:
I think having a url param available to `WORKGROUP_MANAGER` sounds good but I think the resource we're trying specify is an "assessor" so this ought to be `assessorId` perhaps? i.e. I think we're also going to want a list of assessors to implement the "Workflow manager re-allocates an application for assessment" story?

`GET /assessors`

where the response includes a representation of their "qualifications" (authorisation to assess PIPE / women etc)? Not to mention their location...

Reply:
I think this depends on how the workflow manager will go about doing their reassignments.  If it's a case of just viewing all of the assignments for a staff member who is on holiday then probationOfficer as query param makes most sense.  If it's a case of viewing all of them in one go to see which have been pending for too long, probably the filter=all vs allocated would be the way.

Reply:
Or should it perhaps be a filter over all assessments on a variable number of days pending?

Reply:
No query param - will let frontend deal with it